dxr/static_unhashed/css/filter.css

```diff
@@ -58,12 +58,16 @@
     -webkit-box-shadow: 0px 4px 5px 0px rgba(0,0,0,0.25);
     box-shadow: 0px 4px 5px 0px rgba(0,0,0,0.25);
 }
-.sf-select-options .selector-option-label {
+.selector-option-label {
     display: inline-block;
     vertical-align: top;
+    width: 6rem;
+}
+.selector-badges {
     width: 10rem;
+    display: inline-block;
```

Comment:
Hmm, this is no good either. Can we be more advanced than hard-coding widths into things? My CSS is rusty, but it seems like something like flexbox might avail. Or tables, failing all else. There's really nothing wrong with tables.

![screen shot 2016-07-20 at 16 44 16](https://cloud.githubusercontent.com/assets/295816/17002459/5e0b4798-4e99-11e6-9fec-b9bfa28c57d3.png)